### PR TITLE
feat: updates and enhancements

### PR DESCRIPTION
**Pull request by nasbench**

### CODE REVIEW

File: rules/windows/process_creation/proc_creation_win_lolbin_manage_bde.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        - Image|endswith: '\wscript.exe'
+        - OriginalFileName: 'wscript.exe'
+    selection_wscript_cli:
+        CommandLine|contains: 'manage-bde.wsf'
+    selection_parent:
+        ParentImage|endswith:
+            - '\cscript.exe'
+            - '\wscript.exe'
+        ParentCommandLine|contains: 'manage-bde.wsf'
+    filter_cmd:
```
Comment (frack113) [inline rules/windows/process_creation/proc_creation_win_lolbin_manage_bde.yml:31]:
```suggestion
    filter_sel_parent:
```
My idea is to have a name that makes it clear that it is not a global filter but for selection_parent.

File: rules/windows/process_creation/proc_creation_win_malware_emotet.yml [hunk 2]
```diff
@@ -1,4 +1,4 @@
-title: Emotet Process Creation
+title: Potential Emotet Activity
```
Comment (frack113) [inline rules/windows/process_creation/proc_creation_win_malware_emotet.yml:1]: modified date too

File: rules/windows/process_creation/proc_creation_win_nltest_recon.yml [hunk 3]
```diff
@@ -1,12 +1,20 @@
...
     - https://docs.microsoft.com/en-us/previous-versions/windows/it-pro/windows-server-2012-r2-and-2012/cc731935(v=ws.11)
     - https://thedfirreport.com/2021/08/16/trickbot-leads-up-to-fake-1password-installation/
+    - https://thedfirreport.com/2020/10/18/ryuk-in-5-hours/
     - https://book.hacktricks.xyz/windows/basic-cmd-for-pentesters
     - https://research.nccgroup.com/2022/08/19/back-in-black-unlocking-a-lockbit-3-0-ransomware-attack/
+    - https://eqllib.readthedocs.io/en/latest/analytics/03e231a6-74bc-467a-acb1-e5676b0fb55e.html
+    - https://redcanary.com/blog/how-one-hospital-thwarted-a-ryuk-ransomware-outbreak/
 author: Craig Young, oscd.community, Georg Lauenstein
 date: 2021/07/24
 modified: 2023/02/01
```
Comment (frack113) [inline rules/windows/process_creation/proc_creation_win_nltest_recon.yml:20]:
```suggestion
modified: 2023/02/04
```

File: rules/windows/process_creation/proc_creation_win_renamed_netsupport_rat.yml [hunk 4]
```diff
@@ -1,7 +1,7 @@
-title: Execution of Renamed NetSupport RAT
+title: Renamed NetSupport RAT Execution
```
Comment (frack113) [inline rules/windows/process_creation/proc_creation_win_renamed_netsupport_rat.yml:1]: modified too

File: rules/windows/process_creation/proc_creation_win_sc_service_tamper_for_persistence.yml [hunk 5]
```diff
@@ -1,7 +1,7 @@
-title: Modification Of Existing Services For Persistence
+title: Potential Persistence Attempt Via Existing Service Tampering
```
Comment (frack113) [inline rules/windows/process_creation/proc_creation_win_sc_service_tamper_for_persistence.yml:1]: modified too

File: rules/windows/process_creation/proc_creation_win_schtasks_powershell_persistence.yml [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+logsource:
+    product: windows
+    category: process_creation
+detection:
+    selection:
+        ParentImage: 'C:\WINDOWS\System32\svchost.exe'
+        ParentCommandLine|contains|all:
+            - '-k netsvcs'
+            - '-s Schedule'
+        CommandLine|endswith:
```
Comment (frack113) [inline rules/windows/process_creation/proc_creation_win_schtasks_powershell_persistence.yml:27]: The last char can not be `"` or `'` ?

File: rules/windows/process_creation/proc_creation_win_schtasks_reg_loader.yml [hunk 7]
```diff
@@ -4,7 +4,7 @@ status: test
 description: Detects the creation of a schtask that executes a base64 encoded payload stored in the Windows Registry using PowerShell.
 references:
     - https://thedfirreport.com/2022/02/21/qbot-and-zerologon-lead-to-full-domain-compromise/
-author: '@Kostastsale, @TheDFIRReport, slightly modified by pH-T (Nextron Systems)'
+author: pH-T (Nextron Systems), @Kostastsale, @TheDFIRReport
 date: 2022/02/12
 modified: 2022/03/15
```
Comment (frack113) [inline rules/windows/process_creation/proc_creation_win_schtasks_reg_loader.yml:9]:
```suggestion
modified: 2023/02/04
```

File: rules/windows/process_creation/proc_creation_win_susp_copy_lateral_movement.yml [hunk 8]
```diff
@@ -54,6 +54,7 @@ detection:
         CommandLine|contains|all:
             - '\\\\'
             - '$'
+            - '\Sysvol\'
```
Comment (frack113) [inline rules/windows/process_creation/proc_creation_win_susp_copy_lateral_movement.yml:57]: The genreic target is like `\\IP\admin$`
Force to have Sysvol too, breack the logic

File: rules/windows/process_creation/proc_creation_win_schtasks_powershell_persistence.yml (hunk 6 above)
Comment (nasbench) [reply rules/windows/process_creation/proc_creation_win_schtasks_powershell_persistence.yml:27]: It's a specific rule, not a generic one.

File: rules/windows/process_creation/proc_creation_win_lolbin_manage_bde.yml (hunk 1 above)
Comment (nasbench) [reply rules/windows/process_creation/proc_creation_win_lolbin_manage_bde.yml:31]: I will use `selection_filter_cmd` and we will use `selection_filter_` notation for these kind of filters that are not FP

File: rules/windows/process_creation/proc_creation_win_lolbin_createdump.yml [hunk 9]
```diff
@@ -5,9 +5,9 @@ description: Detects uses of the createdump.exe LOLOBIN utility to dump process
 references:
     - https://www.crowdstrike.com/blog/overwatch-exposes-aquatic-panda-in-possession-of-log-4-shell-exploit-tools/
     - https://twitter.com/bopin2020/status/1366400799199272960
-author: Florian Roth (Nextron Systems), Nasreddine Bencherchali
+author: Florian Roth (Nextron Systems), Nasreddine Bencherchali (Nextron Systems)
 date: 2022/01/04
-modified: 2022/08/19
+modified: 2023/02/04
```
Comment (frack113) [inline rules/windows/process_creation/proc_creation_win_lolbin_createdump.yml:10]: no need to update for author change
```suggestion
modified: 2022/08/19
```

**Review by frack113 — APPROVED** [review]

File: rules/windows/process_creation/proc_creation_win_reg_dumping_sensitive_hives.yml [hunk 10]
```diff
@@ -1,15 +1,19 @@
-title: Grabbing Sensitive Hives via Reg Utility
+title: Suspicious Sensitive Hives Dump Via Reg.EXE
 id: fd877b94-9bb5-4191-bb25-d79cbd93c167
+related:
+    - id: 038cd51c-3ad8-41c5-ba8f-5d1c92f3cc1e
+      type: obsoletes
 status: test
-description: Dump sam, system or security hives using REG.exe utility
+description: Detects usage of "reg.exe" in order to dump senstive registry hives, which includes SAM, SYSTEM and SECURITY
```
Comment (phantinuss) [inline rules/windows/process_creation/proc_creation_win_reg_dumping_sensitive_hives.yml:7]:
```suggestion
description: Detects the usage of "reg.exe" in order to dump sensitive registry hives, which includes SAM, SYSTEM and SECURITY
```

File: rules/windows/process_creation/proc_creation_win_reg_dumping_sensitive_hives.yml [hunk 11]
```diff
@@ -1,15 +1,19 @@
-title: Grabbing Sensitive Hives via Reg Utility
+title: Suspicious Sensitive Hives Dump Via Reg.EXE
```
Comment (phantinuss) [inline rules/windows/process_creation/proc_creation_win_reg_dumping_sensitive_hives.yml:1]:
```suggestion
title: Dumping of Sensitive Hives Via Reg.EXE
```

File: rules/windows/process_creation/proc_creation_win_reg_query_registry.yml [hunk 12]
```diff
@@ -1,11 +1,12 @@
-title: Query Registry
+title: Potential Configuration And Service Reconnaissance Via Reg.EXE
 id: 970007b7-ce32-49d0-a4a4-fbef016950bd
 status: test
-description: Adversaries may interact with the Windows Registry to gather information about credentials, the system, configuration, and installed software.
+description: Detects usage of "reg.exe" in order to query reconnaissance information from the registry. Adversaries may interact with the Windows Registry to gather information about credentials, the system, configuration, and installed software.
```
Comment (phantinuss) [inline rules/windows/process_creation/proc_creation_win_reg_query_registry.yml:4]:
```suggestion
description: Detects the usage of "reg.exe" in order to query reconnaissance information from the registry. Adversaries may interact with the Windows registry to gather information about credentials, the system, configuration, and installed software.
```

File: rules/windows/process_creation/proc_creation_win_reg_rdp_keys_tamper.yml [hunk 13]
```diff
@@ -1,12 +1,12 @@
-title: Enabling RDP Service via Reg.exe
+title: Potential Tampering With RDP Related Registry Keys Via Reg.EXE
 id: 0d5675be-bc88-4172-86d3-1e96a4476536
 status: experimental
-description: Detects the execution of reg.exe and subsequent command line arguments for enabling RDP service on the host by tampering with the 'CurrentControlSet\Control\Terminal Server' subkeys
+description: Detects the execution of "reg.exe" for enabling/disabling RDP service on the host by tampering with the 'CurrentControlSet\Control\Terminal Server' values
```
Comment (phantinuss) [inline rules/windows/process_creation/proc_creation_win_reg_rdp_keys_tamper.yml:4]:
```suggestion
description: Detects the execution of "reg.exe" for enabling/disabling the RDP service on the host by tampering with the 'CurrentControlSet\Control\Terminal Server' values
```

File: rules/windows/process_creation/proc_creation_win_reg_service_imagepath_change.yml [hunk 14]
```diff
@@ -1,4 +1,4 @@
-title: Service ImagePath Change with Reg.exe
+title: Suspicious Service ImagePath Value Change Via Reg.EXE
```
Comment (phantinuss) [inline rules/windows/process_creation/proc_creation_win_reg_service_imagepath_change.yml:1]:
```suggestion
title: Changing Existing Service ImagePath Value Via Reg.EXE
```
the image path itself is not suspicious, so I would rephrase the title

File: rules/windows/process_creation/proc_creation_win_renamed_jusched.yml [hunk 15]
```diff
@@ -1,12 +1,12 @@
-title: Renamed jusched.exe
+title: Renamed Jusched.EXE Execution
 id: edd8a48c-1b9f-4ba1-83aa-490338cd1ccb
 status: test
-description: Detects renamed jusched.exe used by cobalt group
+description: Detects execution of a renamed "jusched.exe" as seen used by the cobalt group
```
Comment (phantinuss) [inline rules/windows/process_creation/proc_creation_win_renamed_jusched.yml:4]:
```suggestion
description: Detects the execution of a renamed "jusched.exe" as seen used by the cobalt group
```

File: rules/windows/process_creation/proc_creation_win_renamed_megasync.yml [hunk 16]
```diff
@@ -1,27 +1,24 @@
...
-        OriginalFileName: 'meg.exe'
+    selection:
+        OriginalFileName: 'megasync.exe'
     filter:
-        Image|endswith: '\meg.exe'
-    condition: selection_proc or ( selection_orig and not filter )
+        Image|endswith: '\megasync.exe'
+    condition: selection and not filter
 falsepositives:
     - Software that illegaly integrates MegaSync in a renamed form
```
Comment (phantinuss) [inline rules/windows/process_creation/proc_creation_win_renamed_megasync.yml:23]:
```suggestion
    - Software that illegally integrates MegaSync in a renamed form
```

File: rules/windows/process_creation/proc_creation_win_renamed_netsupport_rat.yml [hunk 17]
```diff
@@ -1,11 +1,12 @@
...
 id: 0afbd410-de03-4078-8491-f132303cb67d
 status: experimental
-description: Detects execution of renamed client32.exe (NetSupport RAT) via Imphash, Product and OriginalFileName strings
+description: Detects execution of a renamed "client32.exe" (NetSupport RAT) via Imphash, Product and OriginalFileName strings
```
Comment (phantinuss) [inline rules/windows/process_creation/proc_creation_win_renamed_netsupport_rat.yml:4]:
```suggestion
description: Detects the execution of a renamed "client32.exe" (NetSupport RAT) via Imphash, Product and OriginalFileName strings
```

File: rules/windows/process_creation/proc_creation_win_renamed_vmnat.yml [hunk 18]
```diff
@@ -1,11 +1,12 @@
-title: Renamed or Portable Vmnat.exe
+title: Renamed Vmnat.exe Execution
```
Comment (phantinuss) [inline rules/windows/process_creation/proc_creation_win_renamed_vmnat.yml:1]: You can remove `filter_portable` as if it is true, `filter_rename` will also be true.

File: rules/windows/process_creation/proc_creation_win_rundll32_process_dump_via_comsvcs.yml [hunk 19]
```diff
@@ -1,10 +1,10 @@
-title: Process Dump via Rundll32 and Comsvcs.dll
+title: Process Dumping Via Comsvcs.DLL
 id: 646ea171-dded-4578-8a4d-65e9822892e3
 related:
     - id: 09e6d5c0-05b8-4ff8-9eeb-043046ec774c
       type: obsoletes
 status: test
-description: Detects process memory dump via comsvcs.dll and rundll32 using different techniques (ordinal, minidump function...etc)
+description: Detects process memory dump via "comsvcs.dll" using rundll32, covering multiple different techniques (ordinal, minidump function...etc)
```
Comment (phantinuss) [inline rules/windows/process_creation/proc_creation_win_rundll32_process_dump_via_comsvcs.yml:7]:
```suggestion
description: Detects a process memory dump via "comsvcs.dll" using rundll32, covering multiple different techniques (ordinal, minidump function, etc.)
```

File: rules/windows/process_creation/proc_creation_win_sc_service_tamper_for_persistence.yml [hunk 20]
```diff
@@ -1,12 +1,12 @@
...
 id: 38879043-7e1e-47a9-8d46-6bec88e201df
 status: test
-description: Detects modification of an existing service on a compromised host in order to execute an arbitrary payload when the service is started or killed as a method of persistence.
+description: Detects modification of an existing service in order to execute an arbitrary payload when the service is started or killed as a potential method for persistence.
```
Comment (phantinuss) [inline rules/windows/process_creation/proc_creation_win_sc_service_tamper_for_persistence.yml:4]:
```suggestion
description: Detects the modification of an existing service in order to execute an arbitrary payload when the service is started or killed as a potential method for persistence.
```

File: rules/windows/registry/registry_set/registry_set_dns_server_level_plugin_dll.yml [hunk 21]
```diff
@@ -1,12 +1,18 @@
...
+title: New DNS ServerLevelPluginDll Installed
 id: e61e8a88-59a9-451c-874e-70fcc9740d67
+related:
+    - id: cbe51394-cd93-4473-b555-edf0144952d9
+      type: derived
+    - id: f63b56ee-3f79-4b8a-97fb-5c48007e8573
+      type: derived
 status: experimental
-description: Detects the installation of a plugin DLL via ServerLevelPluginDll parameter in Registry, which can be used to execute code in context of the DNS server (restart required)
+description: Detects the installation of a DNS plugin DLL via ServerLevelPluginDll parameter in Registry, which can be used to execute code in context of the DNS server (restart required)
```
Comment (phantinuss) [inline rules/windows/registry/registry_set/registry_set_dns_server_level_plugin_dll.yml:9]:
```suggestion
description: Detects the installation of a DNS plugin DLL via ServerLevelPluginDll parameter in registry, which can be used to execute code in context of the DNS server (restart required)
```

File: rules/windows/process_creation/proc_creation_win_dsquery_domain_trust_discovery.yml [hunk 22]
```diff
@@ -0,0 +1,31 @@
+title: Domain Trust Discovery Via Dsquery
+id: 3bad990e-4848-4a78-9530-b427d854aac0
```
[the remark this answers is not in the document]
Comment (nasbench) [reply rules/windows/process_creation/proc_creation_win_dsquery_domain_trust_discovery.yml:2]: Will do

**Review by phantinuss — APPROVED** [review]

**Pull request merged**